book_source/02_demos_tutorials_workflows/04_more_web_interface/02_hidden_analyses.Rmd

```diff
@@ -467,8 +467,37 @@ An example of multi-settings pecan xml file also may look like below:
 ```
 
+### Running SDA on remote
+In general, the idea is that sending, running and monitoring a SDA job should be all done using just two functions (`SDA_remote_launcher` and `Remote_Sync_launcher`). `SDA_remote_launcher` checks for the XML settings defining the run and sets up the SDA on the remote machine and sends a qusb command for running the job. `Remote_Sync_launcher` on the other hand sits on the local machine and monitors the progress of the job and brings back the outputs as long as the job is running. 
+
+`SDA_remote_launcher` sets up the job by copying a template SDA workflow R script as well as a bash file template ready for submission to the remote machine. This functions checks the paths to the all inputs including met, soil, `site_pft` and etc testing whether they exists on the remote machine or not; If they don't it copies them over and replace the settings accordingly. After submitting the bash script it returns the PID of the job writing the log file, allowing the `Remote_Sync_launcher` to monitor the progress of the run and check to see if the job is still alive and if the `sda.output.rdata` is being updated since the last check or not.
+
+`Remote_Sync_launcher` function also follows the progress of the remote job by executing a nohup command on a template R script and keeping the R console open for further use. This R script as it was mentioned above constantly pings the given PID every 5 minutes and copies over the SDA output.
+
+Several points on how to prepare your xml settings for the remote SDA run:
+1- Going through the main pecan workflow.R, if you were able to generate `pecan.TRAIT.xml` your setting is ready to be used for a SDA run. All you need to do extra is adding your state data assimilation tags.
+2- Inside the xml setting `<outdir>` is and will need to be pointing to a local directory where `SDA_remote_launcher` will look for either a `sample.Rdata` file or a `pft` folder.
+3- You need to setup your `<host>` tag accroding to desired remote machine. You can learn more about that on the `Remote execution with PEcAn` section of the documenation. Please make sure that `<folder>` tag inside `<host>` is poitning to a directory where you want to store and run your SDA job. 
+4- Finally make sure the <binary> tag inside the <model> tag is set to the right path on the remote machine.
+
+### Restart functionality in SDA
+In case you prefer to run your SDA analysis in multiple stages where each phase picks up where the previous one left, you can use the `restart` argument in the `sda.enkf.multisite` function. All you need to do is make sure that the output from previous step exists in the `SDA` folder (in the `outfolder`) and the `<start.date>` is the same as `<end.date>` of previous step. When you run SDA with the restart functionality, it loads  the output from previous step in and it uses configs already written in the run folder to sets itself up for the next step. Using the restart argument could be as easy as :
```

Comment:
```suggestion
If you prefer to run your SDA analysis in multiple stages, where each phase picks up where the previous one left off, you can use the `restart` argument in the `sda.enkf.multisite` function. You need to make sure that the output from previous step exists in the `SDA` folder (in the `outfolder`), and the `<start.date>` is the same as `<end.date>` from the previous step. When you run the SDA with the restart parameter, it will load the output from the previous step and use configs already written in the run folder to set itself up for the next step. Using the restart argument could be as easy as :
```